mev-boost/server/mock_relay.go

```diff
@@ -215,25 +214,15 @@ func (m *mockRelay) MakeGetHeaderWithConstraintsResponse(value uint64, blockHash
 	txsRoot := rootNode.Hash()
 
 	bidWithProofs := m.MakeGetHeaderWithProofsResponseWithTxsRoot(value, blockHash, parentHash, publicKey, version, phase0.Root(txsRoot))
-	bidWithProofs.Proofs = make([]*InclusionProof, len(constraints))
 
-	for i, con := range constraints {
-		generalizedIndex := int(math.Pow(float64(2), float64(21))) + i
-
-		proof, err := rootNode.Prove(generalizedIndex)
-		if err != nil {
-			panic(err)
-		}
-
-		merkleProof := new(SerializedMerkleProof)
-		merkleProof.FromFastSszProof(proof)
-
-		bidWithProofs.Proofs[i] = &InclusionProof{
-			TxHash:      con.hash,
-			MerkleProof: merkleProof,
-		}
+	// Calculate the inclusion proof
+	inclusionProof, err := CalculateMerkleMultiProofs(rootNode, constraints)
+	if err != nil {
+		panic(err)
```

Comment:
I know this is old code and for the mocked relay but we should definitely not panic here :)

Reply:
Yeah I took inspo from the old code before. Changed it to an error log, that's fine?